core/drivers/crypto/caam/include/caam_trace.h

```diff
@@ -183,6 +184,29 @@
 #define DMAOBJ_TRACE(...)
 #endif
 
+/* ECC */
+#if CAAM_DBG_TRACE(ECC)
+#define ECC_TRACE DRV_TRACE
+#if CAAM_DBG_DESC(ECC)
+#define ECC_DUMPDESC(desc)                                                     \
+	do {                                                                   \
+		ECC_TRACE("ECC Descriptor");                                   \
+		DRV_DUMPDESC(desc);                                            \
+	} while (0)
+#else
+#define ECC_DUMPDESC(desc)
```

Comment:
Since these macros are supposed to be followed by a `;` it might cause a warning if the macro is empty. It would be more safe to defined these "empty" defines as `do { } while (0)`.
The other already merged defines can be fixed in a separate PR.

Reply:
Ok